docs/rbac.rst

```diff
@@ -0,0 +1,379 @@
+Role Based Access Control
+=========================
+
+.. important::
+
+    This feature is a Tech Preview and may be subject of change.
+
+.. important::
+
+    User ``admin`` automatically gets all RBAC permissions.
+
+RPM plugin presents Role Based Access for these endpoints :
```

Comment:
what about copy, compsxml and rest of the content?

Reply:
Added note about the content and rpm copy.